### PR TITLE
Twilio connector

### PR DESCRIPTION
**Pull request by repoofideas**
Applied black code style and unit test coverage is improved

### CODE REVIEW

File: twilio-connector/twilio_connector.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import argparse
+import pandas as pd
+from ratelimit import limits, sleep_and_retry
+from collections import defaultdict
+from cnvrgv2 import Cnvrg
+from twilio.rest import Client
+
+# The rate limit based on Twilio's rate of dequeueing messages
+# For details, refer to https://support.twilio.com/hc/en-us/articles/115002943027-Understanding-Twilio-Rate-Limits-and-Message-Queues#h_01F0265SAZ9BYNPG0AQZ0AKRHZ
+NUM_OF_CALLS = 3
```
Comment (orionsBeltWest) [inline twilio-connector/twilio_connector.py:12]: Is it possible to pass these global var via config files? 

Okay now I see that these are default func arguments.

File: twilio-connector/twilio_connector.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    #     print(msg.author)
+    #     print(msg.body)
+
+    for msg in messages:
+        messages_map['date_created'].append(msg.date_created)
+        messages_map['author'].append(msg.author)
+        messages_map['body'].append(msg.body)
+
+    df = pd.DataFrame(messages_map)
+    df.dropna(inplace=True)
```
Comment (orionsBeltWest) [inline twilio-connector/twilio_connector.py:125]: Recommend not to drop NA

File: twilio-connector/twilio_connector.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    df.to_csv(args.local_dir+'/'+args.file_name, index=False)
```
Comment (orionsBeltWest) [inline twilio-connector/twilio_connector.py:126]: Probably I will check if the dir is present?

File: twilio-connector/twilio_connector.py [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+    # Store twilio csv as cnvrg dataset
+    if args.cnvrg_dataset.lower() != 'none':
+        cnvrg = Cnvrg()
+        ds = cnvrg.datasets.get(args.cnvrg_dataset)
+        try:
+            ds.reload()
+        except:
```
Comment (orionsBeltWest) [inline twilio-connector/twilio_connector.py:134]: Try to catch a specific exception

File: twilio-connector/twilio_connector.py (hunk 2 above)
Comment (repoofideas) [reply twilio-connector/twilio_connector.py:125]: dropna is removed

File: twilio-connector/twilio_connector.py (hunk 3 above)
Comment (repoofideas) [reply twilio-connector/twilio_connector.py:126]: The code will throw filenotfound error by default, do you want me to try catch it?

File: twilio-connector/twilio_connector.py (hunk 4 above)
Comment (repoofideas) [reply twilio-connector/twilio_connector.py:134]: Added custom NoneCnvrgDatasetError exception

**Pull request closed without merging**